### DISCUSSION TITLE
Access global data from a directory data file

### DISCUSSION THREAD

**Discussion by puleddu** [11ty/eleventy#3030] [Q&A] [answered]
I'm trying to access data that's made available by the Global Data File`_data/snippets.js` from the Javascript Data File `about/about.11tydata.js`. Is it possible? I'm not sure how this could be done. I would have imagined that I could access some global data object from the Javascript file, but I haven't had any luck trying to achieve that. 

Any ideas?

**Comment by puleddu** [comment, marked as answer]
Trying to answer my own question, it seems like the following is a viable option:

`about.11tydata.js`
```
module.exports = {
  eleventyComputed: {
    about: data => data.snippets
  }
};
```
Now, the data in `about` is available for use in my templates, such as `about.md`.
I wonder if it's also the easiest, most efficient way do do something like this.

**Reply by pdehaan** [reply]
Yep, that's how I'd do it.
